packages/terra-filter-view/src/_FilterSelect.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
```

Comment:
When the package name is finalized we should update the file name to match.

`Filter.jsx` / `FilterView.jsx` / `FilterList.jsx` / `SortMaster5000.jsx`

Reply:
This will also influence the scss variable names